packages/core/admin/package.json

```diff
@@ -134,6 +136,7 @@
     "webpack-cli": "4.9.1",
     "webpack-dev-server": "4.7.3",
     "webpackbar": "5.0.0-3",
+    "whatwg-fetch": "3.6.2"
```

Comment:
I think this one can be removed as it is a polyfill and browsers support it natively.

Instead can you remove this from the webpack.alias.js file and remove the dependency int @strapi/helper-plugin package?